inc/contact-page.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * MEOM contact and settings page.
+ *
+ * @package MEOM Dodo
+ */
+
+namespace MEOM\dodo;
+
+/**
+ * Create the contact page.
+ *
+ * @return void
+ */
+function add_contact_admin_menu() {
+    add_options_page(
+        esc_html__( 'MEOM Contact', 'meom-dodo' ),
+        esc_html__( 'MEOM Contact', 'meom-dodo' ),
+        'manage_options',
+        'meom-dodo-contact-page',
+        __NAMESPACE__ . '\meom_contact_page'
+    );
+}
+add_action( 'admin_menu', __NAMESPACE__ . '\add_contact_admin_menu' );
+
+/**
+ * Init settings.
+ *
+ * @return void
+ */
+
+function contact_settings_init() {
+    register_setting( 'meom_dodo_contact', 'meom_dodo_contact_settings' );
+
+    // Register a new section in the "contact" page.
+    add_settings_section(
+        'meom_dodo_contact_section',
+        esc_html__( 'Contact MEOM details', 'meom-dodo' ),
+        __NAMESPACE__ . '\settings_section_callback',
+        'meom_dodo_contact'
+    );
+}
+add_action( 'admin_init', __NAMESPACE__ . '\contact_settings_init' );
+
+/**
+ * Render page description.
+ *
+ * @return void
+ */
+function settings_section_callback() {
+    echo sprintf( esc_html__( 'Contact MEOM using email %s.', 'meom-dodo' ), '<a href="mailto:tech@meom.fi">tech@meom.fi</a>' );
```

Comment:
Is this contact information for our customers? Or for plugin users? Then the mail address should be support@meom.fi

Reply:
All of this is going to change. @SinttiSalminen has copy in place and these will be updated to that.